sql_generators/glean_usage/templates/dau_reporting_clients_daily_v1.query.sql

```diff
@@ -0,0 +1,143 @@
+{{ header }}
+
+WITH base AS (
+  SELECT
+    submission_timestamp,
+    DATE(submission_timestamp) AS submission_date,
+    client_info.client_id,
+    sample_id,
+    normalized_channel,
+    client_info.app_display_version,
+    client_info.app_build,
+    normalized_os,
+    normalized_os_version,
+    client_info.locale,
+    -- metadata.geo.country,
+    {% if has_distribution_id %}
+    metrics.string.metrics_distribution_id AS distribution_id,
+    {% else %}
+    CAST(NULL AS STRING) AS distribution_id,
+    {% endif %}
+    {% if has_profile_group_id %}
+    metrics.uuid.legacy_telemetry_profile_group_id AS profile_group_id,
+    {% else %}
+    CAST(NULL AS STRING) AS profile_group_id,
+    {% endif %}
+    {% if app_name == "firefox_desktop" %}
+    metrics.counter.browser_engagement_uri_count,
+    metrics.counter.browser_engagement_active_ticks,
+    metrics.uuid.legacy_telemetry_client_id,
+    {% endif %}
+    SAFE.PARSE_DATE('%F', SUBSTR(client_info.first_run_date, 1, 10)) AS first_run_date,
+    mozfun.glean.parse_datetime(ping_info.end_time) AS parsed_end_time,
```

Comment:
I found these fields in the dau_reporting (firefox_desktop) table. Should we add these?
```
    document_id, 
    metadata.geo.country,
    metadata.geo.city, 
    metadata.geo.subdivision1 AS geo_subdivision1, 
    metadata.geo.subdivision2 AS geo_subdivision2,
    metadata.geo.db_version AS geo_db_version,
    metadata.isp.name AS isp_name,
    metadata.isp.organization AS isp_organization,
    metrics.string.glean_client_annotation_experimentation_id,
    metrics.uuid.legacy_telemetry_profile_group_id,
```

Reply:
Hey @gkatre thanks for taking a look, as far as I am aware we should not be including those fields in the downstream tables with the exception of `legacy_telemetry_profile_group_id` on Desktop which as far as I can tell already is in the query. My understanding of what we should include is based on the following document:
https://docs.google.com/document/d/1leZN_CUREyfpT3xzI9-6vU5hw52bOkN0Y4Zr67hjXAE/edit?tab=t.0#heading=h.633blik5vmmq

(please refer to the `What should the new ping contain?` section of the document)

Reply:
I see some of the fields mentioned in "Dimensions / Client Information" section: `Assigned on backend: (country, city, sample_id, ISP, submission_date).`
Perhaps just these?
    metadata.geo.country,
    metadata.geo.city, 
    metadata.isp.name AS isp_name

Reply:
Assigned on backend refers to the ingestion pipeline prior to the data landing in BigQuery. I can double check with @irrationalagent if any of those field you mention should also be included.